### PR TITLE
Deduplicate labels from the response for multilabel

### PR DESCRIPTION
**Pull request by rajasbansal**
# Pull Review Summary

## Description

Deduplicate labels for multilabel response from the llm

## Type of change

- Bug fix (change which fixes an issue)

## Tests

locally

### CODE REVIEW

File: src/autolabel/models/vllmmodel.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+import json
```
Comment (rishabh-bhargava) [inline src/autolabel/models/vllmmodel.py:1]: Unrelated change?

**Review by Abhinav-Naikawadi — COMMENTED** [review]
unrelated changes included?

**Review by rishabh-bhargava — APPROVED** [review]
